### ISSUE TITLE
Extension does not work with new versions of eslint

### ISSUE THREAD

**Issue by exsesx** [prettier/prettier-vscode#763]
Prettier cannot resolve the eslint configuration and uses fallback options instead of eslint rules (`prettier.eslintIntegration: true`)

The issue is reproduced using the eslint versions: 
- 5.15.2 
- 5.15.3

The solution for me at the moment is downgrade eslint to 5.15.1

**Comment by rimatla** [comment]
Any updates? Prettier stopped working out of the blue today, I didn't update anything unless VSC updated itself under the hood.

**Comment by icyJoseph** [comment]
We've run into the same issue. Prettier is not applying our elsint rules.

**Comment by through-a-haze** [comment]
Same here. Downgrading eslint doesn't help. I have 
```
"space-before-function-paren": [
      "error",
      {
        "anonymous": "always",
        "named": "never",
        "asyncArrow": "always"
      }
    ]
```
in my eslint rules and it used to work but today Prettier started removing spaces between "function" keyword and parentheses in anonymous functions.

**Comment by Titozzz** [comment]
We started seeing the same issues. Any more info needed to reproduce the issue?

**Comment by bradennapier** [comment]
Same info - extension is dead?

**Comment by through-a-haze** [comment]
It seems to work again for me. Don't know why, didn't do anything.
eslint -v
v5.15.3

prettier -v
1.16.4

**Comment by jhpratt** [comment]
Doesn't appear to work at all for me, disabling prettier until further notice. I imagine others will be doing the same.

**Comment by adrienharnay** [comment]
> Downgrading to 5.15.1 does fix the issue.

As @icyJoseph said, downgrading Eslint to 5.15.1 fixes the issue.

**Comment by muhozi** [comment]
Downgrading doesn't work.
```
eslint -v
v5.15.1
prettier -v
1.16.4
```

**Comment by icyJoseph** [comment]
> Downgrading doesn't work.
> 
> ```
> eslint -v
> v5.15.1
> prettier -v
> 1.16.4
> ```

Perhaps a dependency is forcing a higher version of eslint on your project? What does `yarn why eslint` print? Anyway, the extension should work with any version. Should we investigate what changed from 5.15.1 to 5.15.2, fix it and submit a PR?

**Comment by muhozi** [comment]
I'm using `eslint 5.16.0`(installed globally not in the project locally). In my case, I had this package called Beautify(hookyqr.beautify) and it was the one messing with my prettier.

In addition, other formatting packages may cause prettier not to work as expected as it was mentioned in the prettier vscode README

<img width="887" alt="Screenshot 2019-04-02 at 23 41 15" src="https://user-images.githubusercontent.com/5815755/55434695-d64cda80-55a0-11e9-8629-5975f9a7e0d7.png">

**Comment by jca41** [comment]
Same problem here. After the vscode updated to 1.33.0 (osx) prettier is now applying the fallback rules.
eslint integration is enabled on settings.

**Comment by TimMeade** [comment]
Adding my 2 cents .  osx .   stopped working with vscode 1.33.0

eslint 5.16.0 or 5.15.1 neither works.

**Comment by TimMeade** [comment]
I just downgraded VSCode to Version 1.32.3 (1.32.3) and everything is working again

**Comment by CiGit** [comment]
Have some of you tried with prettier-eslint's CLI with different eslint version ? Trying to narrow down the issue.

**Comment by quentez** [comment]
@CiGit The `prettier-eslint` command works for us with ESLint `5.16.0`. But not in VSCode.

**Comment by bradzacher** [comment]
Unless I'm mistaken, the problem is that this extension bundles its own version of `prettier-eslint` - which means it also has its own version of `eslint@^4.0.0`.

This means that even if you have eslint installed locally, the extension will run the lint and fixers with its local version of eslint 4.
In a lot of cases it'll work, but there are missing things (like support for scoped plugins).

At least that was what I found when I was investigating why [patching in my fix](https://github.com/prettier/prettier-eslint/pull/197) locally didn't fix the extension (I had to go into the vscode extension folder and patch it there for it to work).
